src/Symfony/Bridge/Doctrine/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * changed guessing of DECIMAL to set the `input` option of `NumberType` to string
  * deprecated not passing an `IdReader` to the `DoctrineChoiceLoader` when query can be optimized with a single id field
  * deprecated passing an `IdReader` to the `DoctrineChoiceLoader` when entities have a composite id
+ * added a `ChoiceList` facade to explicit choice lists caching based on callback options or loader when configured by specific form types
```

Comment:
```suggestion
 * added a `ChoiceList` facade for explicit choice lists caching based on callback options or loader when configured by specific form types
```

Reply:
This is not really the same meaning:
"I added something **to explicit** something when being something"
"I added something for **explicit something** when being something"

I would prefer to keep the first.

Reply:
I would ask a native english speaker here :)

Reply:
`explicit` is an adjective, not a verb. So you cannot `to explicit smth`
https://dictionary.cambridge.org/dictionary/english-russian/explicit

Reply:
Indeed. It is in French and wrongly translated as a verb here http://www.wordreference.com/fren/expliciter. I would propose `added a `ChoiceList` facade to make explicit choice lists caching...` then.

Reply:
```suggestion
 * added a `ChoiceList` facade to leverage explicit choice list caching based on callback options or specific loaders
```